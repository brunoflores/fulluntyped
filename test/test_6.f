{x=true, y=false};
{x=true, y=false}.x;
{true, false};
{true, false}.1;
