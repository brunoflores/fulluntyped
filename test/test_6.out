{x=true, y=false} : {x:Bool, y:Bool}
true : Bool
{true, false} : {Bool, Bool}
true
  : Bool
